content/recipes/creamy_tomato_pasta.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "One-Pot Creamy Tomato Pasta"
+tags: ["pasta", "main"]
+---
+
+# One-Pot Creamy Tomato Pasta
+
+**Serves:** 4 (double portion for freezing)  
+**Cook time:** 20 mins
+
+Easy, tasty and very little clean-up! It's a great midweek dinner that also works well as a cold salad the next day if you make extra. We love to serve this with a hunk of crusty bread to clean up the juices left on the plate.
+
+## Ingredients (Double Portion - Cook to Freeze Point)
+
+- 1 onion
+- 3 cloves of garlic
+- 1 aubergine
+- 150g mushrooms of your choice
+- 1½ tablespoons oil
+- 1 tablespoon tamari/soy sauce
+- 100ml water
+- 2 x 400g tins of chopped tomatoes
+- 100g tomato purée
+- 1 tablespoon maple syrup
+- ½ teaspoon freshly ground black pepper
+- 500ml vegetable stock
+- 500ml non-dairy milk (we use oat milk)
+- 500g wholemeal pasta (use gluten-free if you prefer)
+- 2 tablespoons nutritional yeast (optional)
+
+To garnish:
+- a small bunch of fresh basil
+- chilli flakes
+- a little nutritional yeast
+
+## Instructions
+
+1. Peel and finely chop the onion and garlic. Slice the aubergine into small bite-size pieces and finely chop the mushrooms.
```

Comment:
just use much more terse, minimal language throughout, for this line for example
Finely chop onion, garlic, mushrooms. Slice aubergine into small bite-size pieces

Reply:
in fact put cutting instruction with ingredients, e.g 1 onion finely chopped